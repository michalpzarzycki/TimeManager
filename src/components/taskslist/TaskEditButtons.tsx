import React from 'react';
import styles from './TaskEditButtons.module.css';

export default function TaskEditButtons({handleDeletePopup, handleDetailsPopup, handleEdit, task}: any) {
    return(
        <div className={styles.buttons}>
        <div className={styles.details} onClick={() => handleDetailsPopup(task)}></div>
        <div className={styles.edit} onClick={() => handleEdit(task.id)}></div>
        <div className={styles.delete} onClick={() => handleDeletePopup(task.id)}></div>
    </div>
    )
}