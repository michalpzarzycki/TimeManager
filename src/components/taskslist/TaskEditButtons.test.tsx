import React from 'react'
import {shallow} from 'enzyme'
import TaskEditButtons from './TaskEditButtons'

test('Expext to render TaskEditButtons component', () => {
    expect(shallow(<TaskEditButtons />)).toMatchSnapshot()
})
