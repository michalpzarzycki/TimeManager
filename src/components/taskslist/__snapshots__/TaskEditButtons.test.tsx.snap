// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expext to render TaskEditButtons component 1`] = `ShallowWrapper {}`;
